cumulusci/cli/tests/test_cci.py

```diff
@@ -715,9 +714,12 @@ def test_org_list(self, cli_tbl):
             dim_rows=[0, 1],
         )
         persistent_table_call = mock.call(
-            [["Name", "Default", "Username"], ["test2", False, "test2@example.com"]],
+            [
+                ["Name", "Default", "Username", "Expires"],
+                ["test2", False, "test2@example.com", "Unknown"],
```

Comment:
I don't know about this seems suspect.

Reply:
@jofsky it makes sense to me -- the sample OrgConfig created above doesn't have an `expires` key set, so "Unknown" is what we expect to find here.

It would be good to add a second sample OrgConfig that _does_ have an expiration date set, and add a line here to confirm that it gets displayed with the expected values.

Reply:
Ok will add